examples/type_inference/type_inference_visitor_unittest.py

```diff
@@ -0,0 +1,243 @@
+"""Unittest for the type_inference_visitor."""
+
+import unittest
+import sys
+import astroid
+from typing import Tuple, List, Dict
+from astroid import test_utils
+from type_inference_visitor import TypeVisitor, set_const, set_binop, \
+    set_unaryop, set_dict, set_list, set_tuple
+
+
+class SetConstFunctionTest(unittest.TestCase):
+    """testers specifically for the function set_const using single nodes."""
+
+    def test_const_str(self):
+        """testing if the function set_const has the correct output for node
+        type str."""
+        expected = str
+        node = test_utils.extract_node("""a = __('sample_string')""")
```

Comment:
Hi David, I added an assignment for the node 'sample_string' because that's the instruction in `test_utils.extract_node` function, [https://github.com/landscapeio/astroid/blob/master/test_utils.py](url) [line108-113]
`    Expressions:
     To extract arbitrary expressions, surround them with the fake
     function call __(...). After parsing, the surrounded expression
     will be returned and the whole AST (accessible via the returned
     node's parent attribute) will look like the function call was
     never there in the first place.`